superset/css_templates/commands/exceptions.py

```diff
@@ -19,8 +19,8 @@
 from superset.commands.exceptions import CommandException, DeleteFailedError
 
 
-class CssTemplateBulkDeleteFailedError(DeleteFailedError):
-    message = _("CSS template could not be deleted.")
+class CssTemplateDeleteFailedError(DeleteFailedError):
+    message = _("CSS template(s) could not be deleted.")
```

Comment:
To match previous exceptions:

```suggestion
    message = _("CSS templates could not be deleted.")
```